applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -355,33 +363,40 @@ class ExplicitCentralDifferencesScheme
     /**
      * @brief This method initializes some rutines related with the explicit scheme
      * @param rModelPart The model of the problem to solve
+     * @param DoF The current dimention of the problem
      */
-    void InitializeExplicitScheme(ModelPart& rModelPart)
+    void InitializeExplicitScheme(
+        ModelPart& rModelPart,
+        const SizeType DoF = 3
```

Comment:
DoF?
=> this is the dimension isn't it?
=> should not have a default of it is needed

Reply:
DoF is definitely not a good name.

number_of_dofs maybe? also i would not give a default value...